ethereum/package.json

```diff
@@ -1,16 +1,10 @@
 {
-  "scripts": {
-    "deploy": "LEDGERHOSTNAME=$LEDGERHOSTNAME truffle deploy",
-    "test": "LEDGERHOSTNAME=$LEDGERHOSTNAME truffle test"
-  },
-  "dependencies": {
-    "truffle": "^5.1.62",
-    "@truffle/hdwallet-provider": "^1.2.1",
-    "path": "^0.12.7"
+  "scripts" : {
+    "ganache": "$(npm bin)/ganache-cli --chainId 1337",
+    "ganache-w-mnem": "$(npm bin)/ganache-cli --chainId 1337 --mnemonic \"$(cat ~/.secrets_mnemonic_development)\""
```

Comment:
Those lines above will create a local ganache in the container with the contract if we go with the docker-compose.
Will docker-compose adjustment take place in a different PR?

Reply:
Actually, this shall be sorted out in the `ethereum/networks-config.yaml` file using env. var `host` key of deleopment netoworks defined there.
As I mentioned during the standup, I tried to do that by setting `host: $LEDGERHOSTNAME` in relevant places of the netowork config, but unfortunatelly, `brownie` framework does not work is the ganache instance is already started beforehand (url and port properly configured), it works only when brownie framework itself starts the ganache(when it does not find it runnig at configured host & port).
The weird thing about this is, that brownie framework actualy properly detects already runnig ganache at `$LEDGERHOSTNAME` and port configured in `networks-config.yaml` - `I can see on running ganache stdout that brownie is making calls to ganache (= it found it running and started to make calls), but after a few calls I get following error:
```shell
(ethereum) bash-3.2$ LEDGERHOSTNAME=http://127.0.0.1 PRIV_KEY_PATH=~/.secrets_key brownie run deploy_bridge.py
Brownie v1.12.1 - Python development framework for Ethereum

EthereumProject is the active project.
  File "brownie/_cli/__main__.py", line 64, in main
    importlib.import_module(f"brownie._cli.{cmd}").main()
  File "brownie/_cli/run.py", line 43, in main
    network.connect(CONFIG.argv["network"])
  File "brownie/network/main.py", line 48, in connect
    rpc.attach(host)
  File "brownie/network/rpc.py", line 148, in attach
    raise ValueError("No RPC port given")
ValueError: No RPC port given
```

Bellow is the network config - see the `host: $LEDGERHOSTNAME` there for development netorks:


```shell
(ethereum) bash-3.2$ brownie networks import networks-config.yaml true
Brownie v1.12.1 - Python development framework for Ethereum

SUCCESS: Network settings imported from 'networks-config.yaml'
```

```shell
(ethereum) bash-3.2$ brownie networks list true
Brownie v1.12.1 - Python development framework for Ethereum

The following networks are declared:

Ethereum
  ├─Mainnet (Infura)
  │ ├─id: mainnet
  │ ├─chainid: 1
  │ ├─explorer: https://api.etherscan.io/api
  │ └─host: https://mainnet.infura.io/v3/$WEB3_INFURA_PROJECT_ID
  ├─Ropsten (Infura)
  │ ├─id: ropsten
  │ ├─chainid: 3
  │ ├─explorer: https://api-ropsten.etherscan.io/api
  │ └─host: https://ropsten.infura.io/v3/$WEB3_INFURA_PROJECT_ID
  ├─Rinkeby (Infura)
  │ ├─id: rinkeby
  │ ├─chainid: 4
  │ ├─explorer: https://api-rinkeby.etherscan.io/api
  │ └─host: https://rinkeby.infura.io/v3/$WEB3_INFURA_PROJECT_ID
  ├─Goerli (Infura)
  │ ├─id: goerli
  │ ├─chainid: 5
  │ ├─explorer: https://api-goerli.etherscan.io/api
  │ └─host: https://goerli.infura.io/v3/$WEB3_INFURA_PROJECT_ID
  └─Kovan (Infura)
    ├─id: kovan
    ├─chainid: 42
    ├─explorer: https://api-kovan.etherscan.io/api
    └─host: https://kovan.infura.io/v3/$WEB3_INFURA_PROJECT_ID

Ethereum Classic
  ├─Mainnet
  │ ├─id: etc
  │ ├─chainid: 61
  │ ├─explorer: https://blockscout.com/etc/mainnet/api
  │ └─host: https://www.ethercluster.com/etc
  └─Kotti
    ├─id: kotti
    ├─chainid: 6
    ├─explorer: https://blockscout.com/etc/kotti/api
    └─host: https://www.ethercluster.com/kotti

Binance
  ├─BSC
  │ ├─id: BSC
  │ ├─chainid: 56
  │ └─host: https://bsc-dataseed.binance.org/
  └─BSC-test
    ├─id: BSC-test
    ├─chainid: 97
    └─host: https://data-seed-prebsc-1-s1.binance.org:8545/

Development
  ├─Ganache-CLI
  │ ├─id: development
  │ ├─cmd: ./node_modules/.bin/ganache-cli --chainId 1337
  │ └─host: $LEDGERHOSTNAME
  │   ├─accounts: 10
  │   ├─evm_version: istanbul
  │   ├─gas_limit: 12000000
  │   ├─mnemonic: brownie
  │   └─port: 8545
  └─Ganache-CLI (Mainnet Fork)
    ├─id: ganache-mainnet-fork
    ├─cmd: ./node_modules/.bin/ganache-cli --chainId 1
    ├─host: $LEDGERHOSTNAME
    └─timeout: 120
      ├─accounts: 10
      ├─evm_version: istanbul
      ├─fork: mainnet
      ├─gas_limit: 12000000
      ├─mnemonic: brownie
      └─port: 8545
```

Reply:
We will need to resolve this issue later, since I need to land this PR in order to get `brownie` tooling in to unblock depending PRs.